pkg/controller/nodegroup/nodegroup_controller.go

```diff
@@ -281,7 +305,100 @@ func (r *ReconcileNodeGroup) createNodeGroupStack(cfnSvc cloudformationiface.Clo
 				Value: aws.String(eks.Spec.ControlPlane.ClusterName),
 			},
 		},
+		Parameters: parameters,
+	})
+
+	return err
+}
+
+func (r *ReconcileNodeGroup) updateNodeGroupStack(cfnSvc cloudformationiface.CloudFormationAPI, nodegroup *clusterv1alpha1.NodeGroup, eks *clusterv1alpha1.EKS, parameters []*cloudformation.Parameter) error {
+
+	templateBody, err := awsHelper.GetCFNTemplateBody(nodeGroupCFNTemplate, nodeGroupTemplateInput{
+		ClusterName:           eks.Spec.ControlPlane.ClusterName,
+		ControlPlaneStackName: eks.GetControlPlaneStackName(),
+		AMI:                   GetAMI(nodegroup.GetVersion(), eks.Spec.Region),
+		NodeInstanceName:      nodegroup.Name,
+		IAMPolicies:           nodegroup.Spec.IAMPolicies,
+	})
+
+	if err != nil {
+		return err
+	}
+
+	_, err = cfnSvc.UpdateStack(&cloudformation.UpdateStackInput{
+		TemplateBody: aws.String(templateBody),
+		StackName:    aws.String(nodegroup.Name),
+		Capabilities: []*string{aws.String("CAPABILITY_NAMED_IAM"), aws.String("CAPABILITY_IAM")},
+		Tags: []*cloudformation.Tag{
+			{
+				Key:   aws.String("ClusterName"),
+				Value: aws.String(eks.Spec.ControlPlane.ClusterName),
+			},
+		},
+		Parameters: parameters,
 	})
 
 	return err
 }
+
+func parseCFNParameterFromCRD(ngSpec clusterv1alpha1.NodeGroupSpec) []*cloudformation.Parameter {
+	if ngSpec.Instance == nil {
```

Comment:
Minor: Do we have a plan to add more parameters other than instance specific? If not may be we can rephrase this as parseInstanceParameters.

Reply:
I wanted this to reflect the work the function does, Parse the parameters for the CFN Stack  from the CRD.  The naming seems coherent with the output: []*cloudformation.Parameters.